### PR TITLE
kde at icecubeopensource.github.com, simplify setup

### PR DESCRIPTION
**Pull request by jllanfranchi**
* kde module now open source, at github; point to it!
* no need for requirements.txt
* update docs accordingly
* Numba is a requirement now

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,12 +0,0 @@
-pip>=1.8
```
[the remark this answers is not in the document]
Comment (jllanfranchi) [reply requirements.txt]: correct

**Comment by philippeller** [comment]
ok, i give you my blessing :D

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (thehrh) [reply requirements.txt]: the requirements in setup.py check out :+1:

**Comment by ts4051** [comment]
Looks good to me, simplifying the install process would be a huge help. The kde moving out of IceCube svn is a particular help, issues with the svn credentials have tripped up quite a few people in the past.

**Pull request merged**